lookupapi/authentication.py

```diff
@@ -71,7 +72,23 @@ def authenticate(self, request):
         if token is None:
             return None
 
-        return (None, token)
+        # get or create a matching Django user if the token has a subject field, otherwise return
+        # no user.
+        subject = token.get('sub', '')
```

Comment:
I think we should also cut and paste https://github.com/uisautomation/iar-backend/blob/master/assets/tests/test_authentication.py from the IAR backend.

Reply:
done